eq-author/src/components/withEntityEditor/index.js

```diff
@@ -104,8 +128,19 @@ const withEntityEditor = entityPropName => WrappedComponent => {
           onChange={this.handleChange}
           onUpdate={this.handleUpdate}
           onSubmit={this.handleSubmit}
+          enableValidationMessage={this.enableValidationMessage()}
         />
       );
+    };
+
+    render() {
+      const wrappedComponent = this.renderWrappedComponent();
+
+      return focusControl ? (
+        <div onBlur={this.handleLoseFocus}>{wrappedComponent}</div>
```

Comment:
We can't have `withEntityEditor` return DOM elements as we often use it to wrap table rows etc. It was originally intended to pass the functions down so the form could use them. Could we follow that pattern here?

@heofs I believe you may have looked into this around the metadata table

Reply:
After refactor where new page updates even when no change, we no longer need to track focus. I've removed this from withEntityEditor